simon_says/central_node/src/central_node.cpp

```diff
@@ -0,0 +1,180 @@
+#include "central_node.hpp"
+
+namespace {
+int queue_size;
+float central_timeout;
+float ui_scaling[2];
+float joystick_scaling[2];
```

Comment:
scaling and bounds should be packed together in a class, and into a proper datastructure as members of a device. Ie., abstract away the names joystick, quad, turtlebot etc please and put them into an enum or configuration file